src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/entity/BookingHistory.kt

```diff
@@ -84,7 +84,12 @@ class BookingHistory(
  * create and amend cannot have more than one prisoner per booking, but this is future-proofing it.
  */
 
-@Entity
+const val APPOINTMENT_TYPE_COURT_PRE = "VLB_COURT_PRE"
```

Comment:
Really would be better off as an enum for these constants but that would have quite a ripple effect through the code at the moment.